sklearn/neighbors/tests/test_lmnn.py

```diff
@@ -0,0 +1,356 @@
+import numpy as np
+
+from sklearn.model_selection import train_test_split
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_raises
+from sklearn.utils.testing import assert_warns
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_true
+from sklearn import datasets
+from sklearn.neighbors import LargeMarginNearestNeighbor as LMNN
```

Comment:
I would prefer if you avoided the "import as" and used the full name in the code :)